### ISSUE TITLE
Teamspeak 3 monitor command failed to query Server with IPv4 and IPv6

### ISSUE THREAD

**Issue by nero82** [GameServerManagers/LinuxGSM#2658]
Greetings,

My Teamspeak3 server is constantly restarted by the monitor command.
I am using Ubuntu 18.04.3 with LinuxGSM 19.12.5

It seems that the monitor command has a problem with the fact that my Teamspeak 3 server is configured for IPv4 and IPv6.

My ts3server.ini looks something like this:
voice_ip=34.100.42.88,2001:3f3:828:742e::2

When I execute the Monitor command it looks like this:
Monitor ts3server: Querying port: tcp: 34.100.42.8820013f3828742e2:10011

**Comment by dgibbs64** [comment]
You would be the first person I have come across to have configured your server in such a way. Is there a specific reason why you have configured the server to have two IP addresses?

LinuxGSM relys on having a single IP address configured to make it easy to manage various things. Having mulitple IP addresses can currently become problematic and more complicated

**Comment by nero82** [comment]
I'm not quite sure anymore, my server has been configured this way for quite a while and hasn't caused any problems so far. As far as I remember I registered a server nickname for my Teamspeak server.  There was the possibility to specify both IPv4 and IPv6, which I did. I had to realize that my server was not reachable with the IPv6 address and some of my friends could not connect to the server with the nickname. So I looked for this possibility to specify both IP addresses.

From my point of view it would be enough if LGSM would take the first address.
